simbev/simbev_class.py

```diff
@@ -480,9 +481,28 @@ def get_charging_capacity(self, location=None, distance=None, distance_limit=50)
             Returns charging capacity.
         """
 
-        # TODO: check if this destination is used for fast charging
+        if self.power_by_usecase:
+            if use_case == "hpc":
+                if distance > self.distance_threshold_extra_urban:
+                    use_case = "highway_fast"
+                else:
```

Comment:
I wanted to check how exactly the charging by use_case file looks, turns out neither it nor the normal slow charging probabilities are currently in the repo as example files